src/frontend/js/components/Search/index.spec.tsx

```diff
@@ -1,27 +1,28 @@
+import React from 'react';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
```

Comment:
Just a nitpick as it's really the same function, but we generally use the [`act` from react-testing-library](https://testing-library.com/docs/react-testing-library/api#act) for concision (one less import).